### ISSUE TITLE
Php Error

### ISSUE THREAD

**Issue by valtro-one** [Contentify/Contentify#215]
Hello,

When i've gone to open the Impressum page i get the following error: http://prntscr.com/gfpmuw http://prntscr.com/gfpn8a

**Comment by valtro-one** [comment]
@chriskonnertz just an FYI this seems to happen when i create a custom page too.

**Comment by Bambutscha1337** [comment]
i think it is the same problem that i have.

https://github.com/Contentify/Contentify/issues/201

i think it has something to do with this

**Comment by chriskonnertz** [comment]
The problem is that if I follow your steps I do *not* see that error.

**Comment by Bambutscha1337** [comment]
The install.php worked fine, so i thought it couldn't be an SQL Problem. I also have the Tables in my MYSQL Database. I added my log file.
[laravel.txt](https://github.com/Contentify/Contentify/files/1274392/laravel.txt)


In another post you reply with:

> Interesting: The exceptions are not mentioned in the log file. Instead there are a lot of issues with the database. Are you sure that Contentify can connect to the database? The last error is: SQLSTATE[HY000] [1045] Access denied for user 'root'@'localhost' (using password: NO)


The Website ist http://swit.heliohost.org/swit.heliohost.org/swit/public/  if you want to look at it.

Edit: I used the zip file on this hosting website where the laravel data is already unpacked. I found the link in another Issue Thread
https://github.com/Contentify/Contentify/issues/211
